Functions/README.md

```diff
@@ -5,16 +5,22 @@
 Follow the subsequent instructions to develop, debug, unit test, and
 integration test FirebaseFunctions:
 
-```
-$ git clone git@github.com:firebase/firebase-ios-sdk.git
-$ cd firebase-ios-sdk/Functions/Example
-$ pod update
-$ open FirebaseFunctions.xcworkspace
-```
+### Prereqs
+
+- At least CocoaPods 1.6.0
+- Install [cocoapod-generate](https://github.com/square/cocoapods-generate)
+
+### To Develop
+
+- Run `pod gen FirebaseFunctions.podspec`
+- `open gen/FirebaseFunctions/FirebaseFunctions.xcworkspace`
```

Comment:
I'd recommend making the command:
pod gen FirebaseFunctions.podspec --auto-open --gen-directory="gen" --clean

and remove the open command.